### ISSUE TITLE
redis_settings to allow REDIS_URI instead of just RedisSettings

### ISSUE THREAD

**Issue by mpdevilleres** [samuelcolvin/arq#201]
I think it is a nice idea to also allow REDIS_URI instead of just RedisSettings...

here's a use case, I am using arq with fastapi and I have a central config.py file to manage the variables.

```python
# config.py
from starlette.config import Config
from arq.connections import RedisSettings

config = Config('.env')
REDIS_ARQ_HOST = config('REDIS_ARQ_HOST', default='0.0.0.0')
REDIS_ARQ_PORT = config('REDIS_ARQ_PORT', cast=int, default=6379)
REDIS_ARQ_DATABASE = config('REDIS_ARQ_DATABASE', cast=int, default=2)
REDIS_ARQ_SETTINGS = RedisSettings(
    host=REDIS_ARQ_HOST,
    port=REDIS_ARQ_PORT,
    database=REDIS_ARQ_DATABASE
)
```

```python
# worker.py
from app.config import REDIS_ARQ_SETTINGS

class WorkerSettings:
    functions = get_tasks()
    redis_settings = REDIS_ARQ_SETTINGS
    on_startup = startup
    on_shutdown = shutdown
```

```python
# worker.py
from app.config import REDIS_ARQ_SETTINGS

class WorkerSettings:
    functions = get_tasks()
    redis_settings = REDIS_ARQ_SETTINGS
    on_startup = startup
    on_shutdown = shutdown
```

above is ok and will work... but its a good idea to keep the config as plain as possible without any other dependencies on other modules so it is cleaner and easily past to other application.

I am thinking of something like.

```python
# config.py
from starlette.config import Config

config = Config('.env')
REDIS_ARQ_HOST = config('REDIS_ARQ_HOST', default='0.0.0.0')
REDIS_ARQ_PORT = config('REDIS_ARQ_PORT', cast=int, default=6379)
REDIS_ARQ_DATABASE = config('REDIS_ARQ_DATABASE', cast=int, default=2)
REDIS_ARQ_URI = config(
    "REDIS_ARQ_URI",
    default=f'redis://{REDIS_ARQ_HOST}:{REDIS_ARQ_PORT}/{REDIS_ARQ_DATABASE}'
)
```

```python
# worker.py
from app.config import REDIS_ARQ_URI

class WorkerSettings:
    functions = get_tasks()
    redis_settings = REDIS_ARQ_URI
    on_startup = startup
    on_shutdown = shutdown
```

above code config.py is a lot cleaner than the previous.

**Comment by mpdevilleres** [comment]
or maybe add on optional attribute to `RedisSettings` called `redis_uri` where it is translated to different attribute of `RedisSettings` so that we can do as follow.

```python
from arq.connections import RedisSettings
from app.config import REDIS_ARQ_URI

class WorkerSettings:
    functions = get_tasks()
    redis_settings = RedisSettings(uri= REDIS_ARQ_URI)
    on_startup = startup
    on_shutdown = shutdown
```

**Comment by inactivist** [comment]
I use `aioredis.util.parse_url(url:str)` to create a `RedisSettings` instance

[example gist](https://gist.github.com/inactivist/d97aac848cc1bde847592905f62c3b6d).

**Comment by inactivist** [comment]
I've created PR #202 to help provide a slightly cleaner solution.

**Comment by ikhomutov** [comment]
This probably can be closed due to #214 
It is possible to use `redis_settings = RedisSettings.from_dsn(REDIS_ARQ_URI)` now
